src/EventListener/LogRevisionsListener.php

```diff
@@ -451,10 +466,11 @@ private function saveRevisionEntityData($class, $entityData, $revType): void
         if ($class->isInheritanceTypeSingleTable()) {
             $params[] = $class->discriminatorValue;
             $types[] = $class->discriminatorColumn['type'];
-        } elseif ($class->isInheritanceTypeJoined()
-            && $class->name === $class->rootEntityName
+        } elseif (
+            $class->isInheritanceTypeJoined() &&
+            $class->name === $class->rootEntityName
         ) {
-            $params[] = $entityData[$class->discriminatorColumn['name']];
+            $params[] = isset($entityData[$class->discriminatorColumn['name']]) ? $entityData[$class->discriminatorColumn['name']] : $class->discriminatorValue;
```

Comment:
use ternary operation for default value if not set

Reply:
Fixed